todo_app/data/db_items.py

```diff
@@ -0,0 +1,106 @@
+import pymongo
+from todo_app.data.todo_item import TodoItem, TODO_STATUS
+from todo_app.data.db_client import Client
+from bson.objectid import ObjectId
+import datetime
+
+
+def get_items_collection(config):
```

Comment:
No error checking  - didnt really have time and didn't think it would add much value for me sorry!